docs/source/tools/sandbox.rst

```diff
@@ -29,18 +29,19 @@ Here, ``da run sandbox --`` tells the SDK Assistant to run ``sandbox`` from the
 Command-line reference
 **********************
 
-Sandbox requires the names of the input ``.dar`` or ``.dalf`` files as arguments to start.
-The available command line options are listed here::
-
-    -p, --port <value>       Sandbox service port. Defaults to 6865.
-    -a, --address <value>    Sandbox service host. Defaults to binding on all addresses.
-    --dalf                   This argument is present for backwards compatibility. DALF and DAR archives are now identified by their extensions.
-    --static-time            Use static time, configured with TimeService through gRPC.
-    -w, --wall-clock-time    Use wall clock time (UTC). When not provided, static time is used.
-    --no-parity              Disables Ledger Server parity mode. Features which are not supported by the Platform become available.
-    --scenario <value>       If set, the sandbox will execute the given scenario on startup and store all the contracts created by it. Two formats are supported: Module.Name:Entity.Name (preferred) and Module.Name.Entity.Name (deprecated, will print a warning when used).
-    --daml-lf-archive-recursion-limit <value>
-                             Set the recursion limit when decoding DAML-LF archives (.dalf files). Default is 1000
-    <archive>...             Daml archives to load. Either in .dar or .dalf format. Only DAML-LF v1 Archives are currently supported.
-    --help                   Print the usage text
-
+  Usage: sandbox [options] <archive>...
```

Comment:
This looks like you've removed the thing that makes it a code block. Have you previewed it locally to check it looks alright?

Reply:
right, just fixed it